specification/vmware/Microsoft.AVS.Management/clusters.tsp

```diff
@@ -25,34 +24,15 @@ interface Clusters {
     LroHeaders = Azure.Core.Foundations.RetryAfterHeader
   >;
 
-  // This could be modeled with ArmCustomPatchAsync if a 202 response with no body is used instead of a 201.
-  // update is ArmCustomPatchAsync<Cluster, ClusterUpdate>;
-  #suppress "@azure-tools/typespec-azure-core/no-openapi" "DO NOT COPY - TODO migrate to LRO apis"
-  @doc("Update a Cluster")
-  @extension("x-ms-long-running-operation", true)
-  @extension(
-    "x-ms-long-running-operation-options",
-    #{ `final-state-via`: "location" }
-  )
-  @armResourceUpdate(Cluster)
-  @patch
-  update(
-    ...ResourceInstanceParameters<Cluster>,
-
-    @bodyRoot
-    @doc("The cluster properties to be updated.")
-    clusterUpdate: ClusterUpdate,
-  ): ArmResponse<Cluster> | {
-    // @doc("Resource update request created.")
-    ...CreatedResponse;
-
-    ...Azure.Core.Foundations.RetryAfterHeader;
-    ...LocationHeader;
-
-    @bodyRoot
-    @doc("The updated cluster.")
-    cluster: Cluster;
-  } | ErrorResponse;
+  @Azure.Core.useFinalStateVia("location")
+  update is ArmCustomPatchAsync<
+    Cluster,
+    ClusterUpdate,
+    Response = Cluster | ArmResourceCreatedResponse<
```

Comment:
I tried `ArmResponse<Cluster> | ArmResourceCreatedResponse<Cluster>` here, but resulted in additional models in OpenAPI.